Provider/src/FirebirdSql.EntityFrameworkCore.Firebird.Tests/TestUtilities/FakeProvider/FakeDbConnection.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Pomelo Foundation. All rights reserved.
```

Comment:
I don't think fakes are needed here. I'm fine with testing with database right away. I.e. FirebirdClient's tests are 99% integration tests. Yes, it takes more time, but tests the whole stack (well for driver there isn't much of other options).